src/main/java/service/CheckUsersService.java

```diff
@@ -0,0 +1,23 @@
+package service;
+
+import models.User;
+
+import java.util.List;
+
+public class CheckUsersService {
+    private List<User>userList;
+
+    public CheckUsersService(List<User> userList) {
+        this.userList = userList;
+    }
+    public boolean check(User user){
+        boolean isExists = false;
+        for (User U: userList){
```

Comment:
we don't use upper case for local variable naming.
Even more, why not use Stream API here?

Reply:
- [x] used Stream API